packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -20,21 +20,17 @@ export const PolygonGraph = (props: Props) => {
     const {dispatch} = props;
     const {coords, showAngles, showSides, range, snapStep} = props.graphState;
 
+    // TODO(benchristel): can the default set of points be removed here? I don't
+    // think coords can be null.
     const points = coords ?? [[0, 0]];
 
-    const pointsSum = points.reduce(
-        (acc, point) => vec.add(acc, point),
-        [0, 0],
-    );
-    const midpoint =
-        pointsSum && vec.scale(pointsSum, 1 / (points.length ?? 1));
-
     const ref = React.useRef<SVGPolygonElement>(null);
+    const dragReferencePoint = points[0];
```

Comment:
This is a sneaky bugfix.

The bug was that sometimes, selecting an entire polygon and pressing an arrow key would not move the polygon. Our tests for the polygon graph happened to catch it (recall that since we're now using `tab` to select the first movable element in our tests, we're now selecting the entire polygon in the tests instead of selecting the first point)

The bug happened because the `midpoint` of the polygon was, in general, not on a snap step. This meant that trying to move the polygon sometimes caused it to snap back to its original location, effectively canceling the move.

Since the points of a polygon are always on a snap step, using the first point as the reference point fixes this.